docs/source/installation.rst

```diff
@@ -32,7 +32,7 @@ Install
 
 1. Install the package. ::
 
-    $ pip install git+https://github.com/unt-libraries/django-premis-event-service@1.2.5
+    $ pip install git+https://github.com/unt-libraries/django-premis-event-service@1.2.6
```

Comment:
We should probably leave the version at 1.2.5, since we may not be doing a 1.2.6
```suggestion
    $ pip install git+https://github.com/unt-libraries/django-premis-event-service@1.2.5
```